src/main/resources/schema/csaf_json_schema.json

```diff
@@ -83,6 +83,9 @@
           "name"
```

Comment:
I don't really like having this in the root project. We should at least move this to a sub-folder `usaf-schema` and include the build script stuff in there.